src/coreclr/tools/Common/TypeSystem/IL/ILImporter.cs

```diff
@@ -212,7 +215,13 @@ private void FindJumpTargets()
                                 CreateBasicBlock(target);
                             else
                                 ReportInvalidBranchTarget(target);
-                            CreateBasicBlock(_currentOffset);
+                            if ((uint)_currentOffset < (uint)_basicBlocks.Length)
+                                CreateBasicBlock(_currentOffset);
```

Comment:
Do we need a check like this for the other cases in this method (e.g. the non-`_s` variant of these instructions)?

Reply:
and in switch too.